packages/react/src/themes/teams/components/Alert/alertStyles.ts

```diff
@@ -101,71 +101,28 @@ const alertStyles: ComponentSlotStylesInput<AlertProps, AlertVariables> = {
     flexGrow: 1,
   }),
 
-  action: ({ variables: v, props: p }): ICSSInJSStyle => ({
-    height: v.actionSize,
-    minWidth: v.actionSize,
-    color: v.actionColor || 'currentColor',
-    border: 0,
-    borderRadius: v.borderRadius,
-
-    [`& .${teamsIconClassNames.filled}`]: {
-      display: 'none',
-    },
-
-    [`& .${teamsIconClassNames.outline}`]: {
-      display: 'block',
-    },
+  action: ({ variables: v, props: p, theme: { siteVariables } }): ICSSInJSStyle => {
+    const iconFilledStyles = getIconFillOrOutlineStyles(false)
```

Comment:
Can we change the signature to something like this: `getIconFillOrOutlineStyles({outline: false})` so that it will be more obvious what the false is referring to?

Reply:
sounds good, applied, thanks